zipkin-lens/src/components/DependenciesPage/DependenciesPage.tsx

```diff
@@ -0,0 +1,261 @@
+/*
+ * Copyright 2015-2020 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { withRouter, RouteComponentProps } from 'react-router-dom';
+import { t } from '@lingui/macro';
+import { useLingui } from '@lingui/react';
+import { makeStyles, Theme, createStyles } from '@material-ui/core/styles';
+import { Box, Typography, Button, CircularProgress } from '@material-ui/core';
+import { KeyboardDateTimePicker } from '@material-ui/pickers';
+import { MaterialUiPickersDate } from '@material-ui/pickers/typings/date';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSearch } from '@fortawesome/free-solid-svg-icons';
+import moment from 'moment';
+import { History, Location } from 'history';
+
+import ExplainBox from './ExplainBox';
+import TraceJsonUploader from '../Common/TraceJsonUploader';
+import TraceIdSearchInput from '../Common/TraceIdSearchInput';
+import {
+  loadDependencies,
+  clearDependencies,
+} from '../../actions/dependencies-action';
+import RootState from '../../types/RootState';
+import DependenciesGraph from './DependenciesGraph';
+
+// Do require because tsc cannot find the definition of trans.
+// TODO: Give a strict type.
+const { Trans } = require('@lingui/macro');
```

Comment:
I guess this is related to my above comment, I think you can just copy-paste the type